### DISCUSSION TITLE
X-Roads with autoortho 0.5.1

### DISCUSSION THREAD

**Discussion by met4leye** [kubilus1/autoortho#351] [Q&A] [answered]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### question

Since autoortho 0.5.1 X-Roads cannot detect the roads anymore, therefore cant remove them to have satellite only roads. Worked previously just fine, any workaround or idea?

**Comment by marloscaida** [comment, marked as answer]
When you finish loading XP, run the command xroads.bat (may not work for that session, but it will for the next one).
In scenery_pack.ini this line has to be before AO:
SCENERY_PACK Custom Scenery/Xroads/

The content of xroads.bat is:

@echo off

rem show xroads cmd line options
rem xroads.exe -h

rem create log file from output for debugging
rem xroads.exe > xroads.log

rem reduces car speed to 80%
rem xroads.exe -v 80

rem enable left-hand-driving support
rem xroads.exe -l

rem enable left-hand-driving support
rem xroads.exe -l

rem also hide rail tracks
rem xroads.exe -r

rem use xroads default settings
xroads.exe

pause

Regards
carlos

**Reply by met4leye** [reply]
Thanks for your answer carlos, so basically when the flight has loaded in or in the main menu? ☺️

**Comment by marloscaida** [comment]
When you see the airplane in the airport.

**Reply by met4leye** [reply]
Thank you 👍😊
